Makefile

```diff
@@ -51,6 +51,14 @@ test-compile: check-links
 test-compile-changed: os-autoinst/
 	export PERL5LIB=${PERL5LIB_} ; for f in `git diff --name-only | grep '.pm'` ; do perl -c $$f 2>&1 | grep -v " OK$$" && exit 2; done ; true
 
+.PHONY: test-yaml-valid
+test-yaml-valid:
+	export PERL5LIB=${PERL5LIB_} ; tools/test_yaml_valid
+
+.PHONY: test-yamllint
+test-yamllint:
```

Comment:
This would never be called. Did you forget to add it as dependency to another make target?

Reply:
No, I forgot to delete that code during development. Thanks for the catch. For this PR is only using test-yaml-valid.